py/fi_astrosims/client.py

```diff
@@ -104,46 +136,151 @@ def update(self, fields = None, sort = None, sample = None, seed = None, **filte
                 v = str(v)
             self.query[k] = v
 
+        # forces count to update to correct value next time it is called
         self._count = None
-        self._aggs = {}
 
     def makeurl(self, path, **query):
         return makeurl(self.simulation.url, path, dictWith(self.query, **query))
 
     def count(self):
-        """Return the count of objects matching this query."""
+        """
+        :return: (int) Number of objects in the query
+        Can also obtain this value through aggs.
```

Comment:
I think the count in aggs is for non-null fields, which may be different. Maybe not worth mentioning.